### PR TITLE
Enable expeditor

### PR DESCRIPTION
**Pull request by dheerajd-msys**
Signed-off-by: dheerajd-msys <dheeraj.dubey@msystechnologies.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,9 +3,9 @@ cache: bundler
 dist: trusty
 sudo: false
 rvm:
-  - 2.2.7
-  - 2.3.4
   - 2.4.1
+  - 2.3.4
+  - 2.4.2
```
Comment (Vasu1105) [inline .travis.yml]: Can we add versions in order.

**Comment by tas50** [comment]
@Vasu1105 Any chance you can update this for the latest release?

**Comment by dheerajd-msys** [comment]
Hi @tas50, now this is updated for latest release 3.3.0

File: .travis.yml [hunk 2]
```diff
@@ -3,9 +3,9 @@ cache: bundler
 dist: trusty
 sudo: false
 rvm:
-  - 2.2.7
```
Comment (tas50) [inline .travis.yml]: This got a big mangled here

**Comment by btm** [comment]
expeditor didn't fire here https://github.com/chef/knife-google/pull/134. I looked and saw that 'Service Accounts' was missing from collaborators and added it.

I made an empty commit here: https://github.com/chef/knife-google/pull/135 but that didn't fire. There was an upgrade of expeditor around that time, maybe that's why.

Another empty commit here worked: https://github.com/chef/knife-google/pull/136

**Pull request merged**